crates/fiber-lib/src/fiber/channel.rs

```diff
@@ -1164,6 +1179,18 @@ where
                         hold_expire_at,
                     },
                 );
+                let timeout_command =
+                    NetworkActorMessage::new_command(NetworkActorCommand::TimeoutHoldTlc(
```

Comment:
if we added to `pending_notify_settle_tlcs`, we will send timeout in handle fn here:
https://github.com/doitian/fiber/blob/ec37f348ceaff21f2b20b37f35d8f18ad016c12e/crates/fiber-lib/src/fiber/channel.rs#L2770-L2780